src/layout/SummaryAndForm.jsx

```diff
@@ -88,6 +88,7 @@ export class SummaryAndForm extends React.Component {
       formSubmittedActionCreator: [
         () => this.props.patient && this.props.patient.uuid && visitActions.fetchPatientActiveVisit(this.props.patient.uuid, this.props.sessionLocation.uuid, ACTIVE_VISITS_REP),
         () => this.props.patient && this.props.patient.uuid && patientActions.getIC3PatientScreeningData(this.props.patient),
+        () => this.props.patient && this.props.patient.uuid && patientActions.getIC3PatientNutritionHistory(this.props.patient),
```

Comment:
@rhenshaw56  do we need to add this here?  I would think we'd only want/need to fetch the Nutrition History from the ClinicianSummary or NutritionSummary pages?